assembly/pom.xml

```diff
@@ -31,6 +31,7 @@
   <packaging>pom</packaging>
 
   <properties>
+    <parquet.deps.scope>provided</parquet.deps.scope>
```

Comment:
where do we use this property?

Reply:
At least `make-distribution.sh` needs this property.
If we don't add this property, `parquet-*-1.10.1.jar` will be in the `${SPARK_HOME}/jars` directory.